OSDOCS-16960 AUTH-2 CQA Part 2 (#113538)
openshift/openshift-docs: modules/authentication-prometheus-system-metrics.adoc

[id="authentication-prometheus-system-metrics_{context}"]
= Authentication metrics for Prometheus

[role="_abstract"]

> {product-title} captures Prometheus metrics that track authentication attempts and outcomes for both the CLI and web console, enabling administrators to monitor login activity and troubleshoot authentication failures.

Reviewer: We're  supposed to get away from feature-focused wording to user-centric wording. And you can create a lead-in sentence for the list:
```suggestion
You can use Prometheus metrics to monitor login activity and troubleshoot authentication failures. 

{product-title} captures the following Prometheus metrics that track authentication attempts and outcomes for both the CLI and web console:
```
